core/src/main/java/com/scalar/db/transaction/jdbc/JdbcTransactionManager.java

```diff
@@ -109,6 +109,16 @@ public DistributedTransaction begin(String txId) throws TransactionException {
     }
   }
 
+  @Override
+  public DistributedTransaction beginReadOnly() {
+    throw new UnsupportedOperationException("implement later");
+  }
+
+  @Override
+  public DistributedTransaction beginReadOnly(String txId) {
+    throw new UnsupportedOperationException("implement later");
+  }
```

Comment:
I will add an implementation for JDBC transactions in a separate PR after this PR is merged.